src/test/java/org/gridsuite/filter/server/ExpertFilterUtilsTest.java

```diff
@@ -1,560 +0,0 @@
-
-        CombinatorExpertRule orFilter = CombinatorExpertRule.builder().combinator(CombinatorType.OR).rules(orRules2).build();
-
-        boolean result = orFilter.evaluateRule(gen);
-
-        assertTrue(result);
-    }
-
-    @Test
-    void testEvaluateExpertFilterWithFalseORCombination() {
-        List<AbstractExpertRule> orRules1 = new ArrayList<>();
-        NumberExpertRule numRule1 = NumberExpertRule.builder().value(0.0)
-                .field(FieldType.MIN_P).operator(OperatorType.GREATER).build(); // false
-        orRules1.add(numRule1);
-        NumberExpertRule numRule2 = NumberExpertRule.builder().value(105.0)
-                .field(FieldType.MAX_P).operator(OperatorType.GREATER_OR_EQUALS).build(); // false
-        orRules1.add(numRule2);
-        NumberExpertRule numRule3 = NumberExpertRule.builder().value(-500.0)
-                .field(FieldType.MIN_P).operator(OperatorType.LOWER).build(); // false
-        orRules1.add(numRule3);
-        NumberExpertRule numRule4 = NumberExpertRule.builder().value(-500.5)
-                .field(FieldType.MAX_P).operator(OperatorType.LOWER_OR_EQUALS).build(); // false
-        orRules1.add(numRule4);
-        CombinatorExpertRule orCombination = CombinatorExpertRule.builder().combinator(CombinatorType.OR).rules(orRules1).build();
-
-        List<AbstractExpertRule> orRules2 = new ArrayList<>();
-        orRules2.add(orCombination);
-        NumberExpertRule numRule5 = NumberExpertRule.builder().value(25.0) // False
-                .field(FieldType.TARGET_V).operator(OperatorType.EQUALS).build();
-        orRules2.add(numRule5);
-        EnumExpertRule enumRule4 = EnumExpertRule.builder().value("OTHER")
-                .field(FieldType.ENERGY_SOURCE).operator(OperatorType.EQUALS).build(); //false
-        orRules2.add(enumRule4);
-        EnumExpertRule enumRule5 = EnumExpertRule.builder().value("HYDRO")
-                .field(FieldType.ENERGY_SOURCE).operator(OperatorType.NOT_EQUALS).build(); //false
-        orRules2.add(enumRule5);
-        StringExpertRule stringRule5 = StringExpertRule.builder().value("TEST")
-                .field(FieldType.ID).operator(OperatorType.BEGINS_WITH).build(); //false
-        orRules2.add(stringRule5);
-        BooleanExpertRule booleanRule6 = BooleanExpertRule.builder().value(false)
-                .field(FieldType.VOLTAGE_REGULATOR_ON).operator(OperatorType.EQUALS).build(); //false
-        orRules2.add(booleanRule6);
-        BooleanExpertRule booleanRule7 = BooleanExpertRule.builder().value(true)
-                .field(FieldType.VOLTAGE_REGULATOR_ON).operator(OperatorType.NOT_EQUALS).build(); //false
-        orRules2.add(booleanRule7);
-
-        CombinatorExpertRule orFilter = CombinatorExpertRule.builder().combinator(CombinatorType.OR).rules(orRules2).build();
-
-        boolean result = orFilter.evaluateRule(gen);
-
-        assertFalse(result);
-    }
-
-    @Test
-    void testEvaluateExpertFilterIgnoreCase() {
```

Comment:
Same

Reply:
Idem, rule tree is tested by https://github.com/gridsuite/filter-server/pull/90/files#diff-b4e0eb7fb6e066e9822531fb175401d400663c91679b6a7c2a936fe2fd7e74ccR48

Reply:
This case is still not tested

Reply:
DONE by https://github.com/gridsuite/filter-server/pull/90/commits/0fef89ec3a4c457b5aa7e85f8169ade69acafe1c#diff-b87a5db1db9cfacd1f77b13c9cb1c86fa26edd75272b6cb552f7df13fe0a50b7R104-R107